web/src/components/dashboard/QuickActionTile.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { QuickActionTile } from "@/components/dashboard/QuickActionTile";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { mockPush, useMockRouter } from "@/test/mock/mockRouter";
+import * as materialUi from "@mui/material";
+import { useMediaQuery } from "@mui/material";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+jest.mock("@mui/material", () => mockMaterialUI());
+
+function mockMaterialUI(): typeof materialUi {
+  return {
+    ...jest.requireActual("@mui/material"),
+    useMediaQuery: jest.fn(),
+  };
+}
+
+const setMobile = (value: boolean): void => {
+  (useMediaQuery as jest.Mock).mockImplementation(() => {
+    return value;
+  });
+};
+
+describe("<QuickActionTile />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRouter({});
+  });
+
+  it("displays name", () => {
+    render(<QuickActionTile name="some name" url="" index={0} />);
+    expect(screen.getByText("some name")).toBeInTheDocument();
+  });
+
+  it("routes to actions/url when clicked", () => {
+    render(<QuickActionTile name="some name" url="some-url" index={0} />);
+    fireEvent.click(screen.getByText("some name"));
+    expect(mockPush).toHaveBeenCalledWith(`${ROUTES.quickActions}/some-url`);
+  });
+
+  it("sets right padding for even-indexed tile when desktop", () => {
+    setMobile(false);
+    render(<QuickActionTile name="some name" url="some-url" index={0} />);
+    expect(screen.getByTestId("quick-action-0")).toHaveStyle("padding: 0px 0.25rem 0px 0px;");
+  });
+
+  it("sets left padding for odd-indexed tile when desktop", () => {
+    setMobile(false);
+    render(<QuickActionTile name="some name" url="some-url" index={1} />);
+    expect(screen.getByTestId("quick-action-1")).toHaveStyle("padding: 0px 0px 0px 0.25rem;");
+  });
+
+  it("sets top padding when mobile", () => {
+    setMobile(true);
+    render(<QuickActionTile name="some name" url="some-url" index={0} />);
+    expect(screen.getByTestId("quick-action-0")).toHaveStyle("padding: 0.25rem 0px 0px 0px;");
+  });
```

Comment:
A bit strange to be doing things as styles, also having tests for them? Do we want to test styles in some cases now?

Reply:
I tested the styles here because there's logic in when they get applied. You know, same as how we'd test an href or something.

Reply:
Yeah makes, sense, I saw the test first and made the comment before seeing the implementation